MantidPlot/src/ConfigDialog.cpp

```diff
@@ -715,8 +715,8 @@ void ConfigDialog::initMantidPage() {
 
   // Populate boxes
   auto faclist = cfgSvc.getFacilityNames();
-  for (auto it = faclist.begin(); it != faclist.end(); ++it) {
-    facility->addItem(QString::fromStdString(*it));
+  for (auto it : faclist) {
```

Comment:
In this and other loops below you probably can use `const auto &`. It's not going to do any difference here, but I'd say it's good practice in general to try to use all those specifiers whenever possible.